.dockerignore

```diff
@@ -0,0 +1,17 @@
+.DS_Store
+build
+docs/_build
+__pycache__
+*.swp
+node_modules
+.project
+yarn.lock
+extra/
```

Comment:
I think someone might want this included, it's whole point is to be used by the build process. Thoughts?

Reply:
Should `extra/` be available in the container? If it is a build artifact, I would leave it out and let the build process create it.

Reply:
No it's not a build artifact, it's a requirement.

Reply:
Well technically both... because artifacts are added to the modules in place to make committing the distributables to GitHub super easy.

Reply:
So I will remove `yarn.lock` and `extra/` from the `.dockerignore`. Would that make this PR mergeable? (also I am resolving the other related conversation in lieu of this one)